universal-application-tool-0.0.1/app/services/program/Predicate.java

```diff
@@ -1,11 +1,14 @@
 package services.program;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
 
 /** String representation of a predicate with variables from the core data model. */
 @AutoValue
 abstract class Predicate {
```

Comment:
suggestion (non-blocking): `public` visibility for `Predicate`

Reply:
done